Probesys/bileto
# imp: Improve the logo with a bolder smile

Related to https://github.com/Probesys/bileto/pull/177/commits/5dbebde4705d0a2f4009e60bc07f3a2651de4188

Changes proposed in this pull request:

- Change the logo with the one with a bolder smile

How to test the feature manually:

1. Refresh and check the smile in the logo is visible when connected

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
